src/ViewModel/SeeMoreLink.php

```diff
@@ -12,13 +12,14 @@ final class SeeMoreLink implements ViewModel
     use ArrayFromProperties;
 
     protected $name;
-
     protected $url;
+    protected $ariaLabel;
 
     public function __construct(Link $link, bool $isInline = false)
     {
         $this->name = $link['name'];
         $this->url = $link['url'];
```

Comment:
If link has url as optional but SeeMoreLink requires it then we should trigger an error in this function if `$link['url']` is not set.

Reply:
We could pair on this on Monday. Or, if we want to get this in today. We could revert my changes and then create a ticket to address this as a followup.

Reply:
I created a test to check the required URL. Can you check it when you have time?